kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -233,6 +233,15 @@
   private static final String TABLE_CREATE_DOC =
           "Automatically create BigQuery tables if they don't already exist";
 
+  public static final String DISABLE_BEST_EFFORT_DEDUPLICATION_CONFIG =         "disableBestEffortDeduplication";
```

Comment:
Do you think `enableBestEffortDeduplication` or even just `bestEffortDeduplication` might be more intuitive, with a default of `true` instead of `false`, might be more intuitive and avoid a double negative?

Reply:
agreed, went ahead and used bestEffortDeduplication